### PR TITLE
Use epsilon for subtext color

### PR DESCRIPTION
**Pull request by mikehenrty**
Based on UI-review from Amy.

### CODE REVIEW

**Comment by mikehenrty** [comment]
@wilsonpage r?

**Comment by DouglasSherk** [comment]
r+

File: gaia-sub-header.js [hunk 1]
```diff
@@ -54,7 +54,7 @@ module.exports = component.register('gaia-sub-header', {
       font-weight: normal;
 
       color:
-        var(--color-zeta);
+        var(--color-epsilon);
```
Comment (wilsonpage) [inline gaia-sub-header.js:57]: This should be changed to use the more abstract colors defined in gaia-theme. For example something like `--text-color` or `--text-color-minus`.

Comment (mikehenrty) [inline gaia-sub-header.js:57]: @wilsonpage That's good! So what Amy is looking for here is actually `title-color`. But I decided to follow the color convention already there. In any case, I'll submit a new PR.

**Pull request merged**